sub-packages/bionemo-amplify/tests/bionemo/amplify/test_hf_rotary.py

```diff
@@ -23,6 +24,7 @@
 from bionemo.amplify.model import AMPLIFYConfig
 
 
+@pytest.mark.skip(reason="Skipping test due to xformers requirement")
```

Comment:
can you create an issue in github and attach it to this skip mark?

Reply:
`
@pytest.mark.skipif(
    not sys.modules.get("xformers"),
    reason="AMPLIFY golden value tests require xformers.",
)
`
is good workaround as well